### ISSUE TITLE
Multiple citation highlighting isn't implemented

### ISSUE THREAD

**Issue by jtkiley** [area/language-latex#14]
The single citation commands and keys look fine, but the multiple citation commands do not (e.g. `\autocites{}`).

I took a look to see if it looked like an easy fix, but it wasn't obvious to me. That's not too surprising, as I haven't had time to dig into really understanding how language packages and the regexes work.

**Comment by area** [comment]
I don't believe it's possible for the last example to  be highlighted correctly, as it involves an unknown number of groups, which isn't something that regex can deal with. If I'm wrong, do let me know.

I've also not introduced the capitalised versions - in my experience LaTeX commands are lowercase by convention. If enough people object to that though, I'll revisit.

**Comment by jtkiley** [comment]
The capitalized versions are defined in the package as a way of handling capitalization of name prefixes (e.g. when an cited work has an author last name with a prefix that is lower case by convention but needs to be capitalized for style or sentence beginning reasons). See the pdf I linked above, pp. 93-101.

There are also other capitalized commands in the package, too.

**Comment by area** [comment]
Ah, apologies, I hadn't appreciated that the capitalised versions of these commands provided distinct functionality.

**Comment by yudai-nkt** [comment]
> an unknown number of groups, which isn't something that regex can deal with

I think we can parse `\autocites`' syntax by the following regex ([this](https://regex101.com/r/G1r9ne/1) is the result):

```javascript
/(\\[aA]utocites)(?:(\()([^)]*)(\))){0,2}(?:(?:(\[)([^\]]*)(\])){0,2}(\{)([^}]*)(\}))+/
```

However, this regex doesn't highlight the syntax properly, which lead me to open a [question](https://discuss.atom.io/t/how-to-define-a-grammar-for-the-group-repeated-by-quantifier/34189) at Atom's forum. After we get some reaction there, we can decide whether it is possible or not.

**Comment by jtkiley** [comment]
@area: No worries. Thanks for your work on this.

**Comment by alexg-k** [comment]
I want to use \cite{key1, key2} and this also breaks syntax highlighting from this point on.

Package version 1.2.0
Atom version 1.52.0
